src/tool/core/mod.rs

```diff
@@ -18,6 +20,7 @@ pub fn run(cli: &Cli) -> anyhow::Result<()> {
 
     match &cli.command {
         cli::Commands::Generate(args) => do_generate(args),
+        &cli::Commands::Copy => copy(),
```

Comment:
Is there a reason we need `&` here?

```suggestion
        cli::Commands::Copy => copy(),
```

Reply:
IDK, that's just what the LSP autofilled for me, probably better to remove that